lib/rules/aria-allowed-role.json

```diff
@@ -0,0 +1,18 @@
+{
+	"id": "aria-allowed-role",
+	"excludeHidden": false,
+	"selector": "[role]",
+	"tags": [
+		"cat.aria",
+		"best-practice"
+	],
+	"metadata": {
+		"description": "Ensure role attribute has a value appropriate for the element",
```

Comment:
Let's change this to "Ensures role attribute has an appropriate value for the element" to be consistent with #1003.